cluster-provision/gocli/cmd/utils/images.go

```diff
@@ -4,5 +4,5 @@ const (
 	// NFSGaneshaImage contains the reference to NFS docker image
 	NFSGaneshaImage = "docker.io/janeczku/nfs-ganesha@sha256:17fe1813fd20d9fdfa497a26c8a2e39dd49748cd39dbb0559df7627d9bcf4c53"
 	// DockerRegistryImage contains the reference to docker registry docker image
-	DockerRegistryImage = "quay.io/libpod/registry:2.7"
+	DockerRegistryImage = "quay.io/libpod/registry:2.8.2"
```

Comment:
Thanks for this but why is this needed for s390x ? We should try to minimize our changes here as the PR is already quite big.